client/helpers/scenes.ts

```diff
@@ -337,8 +332,46 @@ export async function createSceneAndSceneItemsEXPERIMENTAL({
   let batchQueuedOps: QueuedBatchOps[] = [];
   let batchErrors: QueuedBatchOps[] = [];
   let sceneItemErrors: SceneItemError[] = [];
-  for (let depth = 0; depth < createSceneItemReqs.length; depth++) {
-    const createItemReqs: CreateSceneItemRequest[] = createSceneItemReqs[depth];
+
+  const reqMap: Map<string, CreateSceneItemRequest[]> = new Map();
+  let nextChildren: CreateSceneItemRequest[] = [];
+  reqMap.set('', nextChildren);
+
+  // create parent map and set ordinals based on request order
+  createSceneItemReqs.forEach((req) => {
```

Comment:
It looks like this logic appears in two places and could be consolidated into a named function.

Reply:
The second instance of that logic has been removed.